src/ui/screen_process.c

```diff
@@ -46,26 +62,39 @@ static component_t* _get_waiting_screen(void)
     return waiting_screen;
 }
 
-static void _screen_process(bool (*is_done)(void), void (*on_timeout)(void), const uint32_t timeout)
+/**
+ * \param is_done WTF
```

Comment:
:thinking: 

not sure this comment helps  :laughing:

Reply:
Comment? Which comment? :D